x-pack/plugins/siem/public/pages/overview/summary.tsx

```diff
@@ -52,7 +47,7 @@ export const Summary = pure(() => (
           defaultMessage="If you have input or suggestions regarding your experience with SIEM, please feel free to {feedback}."
```

Comment:
Can we change "SIEM" to "Elastic SIEM" in this sentence.  So it reads: "If you have input or suggestions regarding your experience with Elastic SIEM, please feel free to {feedback}."